src/main/java/nextstep/subway/path/application/PathService.java

```diff
@@ -0,0 +1,48 @@
+package nextstep.subway.path.application;
+
+import nextstep.subway.line.application.LineService;
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.path.domain.Path;
+import nextstep.subway.path.domain.PathFinder;
+import nextstep.subway.path.dto.PathResponse;
+import nextstep.subway.station.application.StationService;
+import nextstep.subway.station.domain.Station;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Objects;
+
+@Service
+public class PathService {
+    private final LineService lineService;
+    private final StationService stationService;
+
+    public PathService(LineService lineService, StationService stationService) {
+        this.lineService = lineService;
+        this.stationService = stationService;
+    }
+
+    public PathResponse findPath(Long startStationId, Long endStationId) {
```

Comment:
```suggestion
    @Transactional(readOnly = true)
    public PathResponse findPath(Long startStationId, Long endStationId) {
```

조회만 하는 기능의 경우 `readOnly` 옵션을 사용해볼 수 있겠네요 :)